### PR TITLE
Change some wording about resources that need to be provided by UPI

### PR DESCRIPTION
**Pull request by sreber84**
Adjusting "The control plane and compute machines that make up the cluster" with "The underlying compute infrastructure" as this wording seems to make more accurate.

### CODE REVIEW

File: modules/installation-overview.adoc [hunk 1]
```diff
@@ -179,7 +179,7 @@ the cluster to the infrastructure that you provided.
 If you do not use infrastructure that the installation program provisioned, you must manage
 and maintain the cluster resources yourself, including:
 
-* The control plane and compute machines that make up the cluster
+* The underlying compute infrastructure
```
Comment (kalexand-rh) [inline modules/installation-overview.adoc:182]: @abhinavdahiya, do you have an opinion on this suggested terminology change?

Comment (abhinavdahiya) [reply modules/installation-overview.adoc:182]: imo this is confusing as users might think the compute machines for cluster are user managed but the control plane is installer created/managed. I like the previous one over this suggested change as it was more explicit

Comment (kalexand-rh) [reply modules/installation-overview.adoc:182]: I agree. Thank you!

**Comment by kalexand-rh** [comment]
@sreber84, thanks for suggesting the change, but I'm going to follow Abhinav's suggestion and retain the original wording.

File: modules/installation-overview.adoc (hunk 1 above)
Comment (sferich888) [reply modules/installation-overview.adoc:182]: Look at this documentation from this perspective. 

Customer Question: 
"When using user-provisioned infrastructure installation you do not use infrastructure that the installation program provisioned but you must manage and maintain the cluster resources yourself, including: what?"

If we say, the "The control plane and compute machines that make up the cluster", what does this mean for "maintain the cluster resources yourself", updates, etc? 

By saying the "underlying compute infrastructure", it make it clear to the user, that "When using user-provisioned infrastructure installation you do not use infrastructure that the installation program provisioned but you must manage and maintain the" - "underlying compute infrastructure", not necessarily the series that run on these hosts. 

If you don't like the proposed language, what language do you suggest, to improve on the existing customer confusion? Simply leaving the language as is; might mean we need to re-write the rest of the section.

Comment (vikram-redhat) [reply modules/installation-overview.adoc:182]: @abhinavdahiya Could you look at Eric's comment? If you don't agree, @kalexand-rh can you close it and the associated BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1854964

Comment (kalexand-rh) [reply modules/installation-overview.adoc:182]: @sferich888, I had a chat with Abhinav. Would "The hosts for the control plane and compute machines that make up the cluster" be an acceptable clarification?

Comment (sferich888) [reply modules/installation-overview.adoc:182]: @sreber84 I think it's better, what's your comment? What the objection with the proposed wording?

Comment (kalexand-rh) [inline modules/installation-overview.adoc:182]: @sreber84, will you PTAL at this suggested change?

```suggestion
* The hosts for the control plane and compute machines that make up the cluster
```

If you're ok with that wording, I'm happy to merge it.

Comment (sreber84) [reply modules/installation-overview.adoc:182]: @kalexand-rh that sounds OK to me and I'm fine with applying this change.

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL at this change?

**Comment by jianlinliu** [comment]
Hmm, "The hosts for the control plane and compute machines", for me, I feel like it need some virtualization hosts for that.  If user is running UPI installation on public cloud, such as: aws/gcp/azure, I am not sure "The hosts" is good wording here. How about "he underlying infrastructure for the control plane and compute machines"?

**Comment by kalexand-rh** [comment]
@jianlinliu, that sounds fine to me, and I've updated the PR. If no one else objects to the update, I'll merge.

**Comment by jianlinliu** [comment]
LGTM.

**Pull request merged**